module-4/backend/week_three.md

```diff
@@ -5,15 +5,87 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+
+  Node is a JavaScript compiler that allows JavaScript to be run independently of the browser.
+
 2. What is Express? What is Express similar to in the Ruby world?
+
+  Express is a web framework, similar to rails, that comes pre-built with a templating language (jade), router functionality, and some essential libraries.
```

Comment:
Yep, I would say it's more similar to Sinatra because it is so lightweight and non-opinionated!